new-examples/jdbc-atomic-batch-execute-operation/jdbc_atomic_batch_execute_operation.out

```diff
@@ -0,0 +1,18 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file and execute the `bal run` command below.
+bal run jdbc_atomic_batch_execute_operation.bal
```

Comment:
I think, this is not the correct way to run this sample. Instead of this, we need to create the ballerina project and run it. WDYT?

Reply:
Yes. In order to run the BBE. I had to create a ballerina project. then `bal run` is sufficient to run the BBE. @daneshk shall I remove the bal file argument in here?

Reply:
then I guess, we need to mention it here to create a Ballerina project and update Ballerina.toml etc

Reply:
Shall we update the `.out` file like https://github.com/ballerina-platform/ballerina-distribution/blob/master/examples/grpc-simple/grpc_simple_service_client.out

Reply:
Updated in https://github.com/ballerina-platform/ballerina-distribution/pull/1751/commits/38b777f53c95cec89610aa9df92d60f28efd9518